libmscore/arpeggio.cpp

```diff
@@ -23,6 +23,15 @@
 
 namespace Ms {
 
+const std::array<const char*, 6> Arpeggio::arpeggioTypeNames = {
+      "Arpeggio",
+      "Up Arpgeggio",
+      "Down arpeggio",
+      "Bracket arpeggio",
+      "Up arpeggio straight",
+      "Down arpeggio straight"
```

Comment:
Make it  `QT_TRANSLATE_NOOP("Palette", "Arpeggio"),` etc. to have them translatable